client/src/locale/en.js

```diff
@@ -916,73 +916,73 @@ const en = {
-        service_requestTooltip: "The URL to request a connection with the service by a member of a collaboration",
+        service_request: "Link for application request",
+        service_requestTooltip: "The URL to request a connection with the application by a member of a collaboration",
         service_requestError: "Requires a valid URI and collaborations must be allowed to connect without approval",
         description: "Description",
-        descriptionPlaceholder: "The description of the service",
+        descriptionPlaceholder: "The description of the application",
         address: "Address",
-        addressPlaceholder: "The address of the service",
+        addressPlaceholder: "The address of the application",
         identity_type: "Identity type",
-        identity_typePlaceholder: "The identity type of the service",
-        identity_typeTooltip: "The primary way of identification for this service",
+        identity_typePlaceholder: "The identity type of the application",
+        identity_typeTooltip: "The primary way of identification for this application",
         uri: "Login URL for users",
-        uriPlaceholder: "The URL where users log in to the service",
-        uriTooltip: "URL where users can log in to your service, displayed prominently to collaboration members.<br/><br/>" +
+        uriPlaceholder: "The URL where users log in to the application",
+        uriTooltip: "URL where users can log in to your application, displayed prominently to collaboration members.<br/><br/>" +
             "These variables will be replaced with the appropriate values: <em>{CO_short_name}</em> and <em>{username}</em>.",
         infoUri: "Website",
-        infoUriPlaceholder: "The URL with information about this service",
-        infoUriTooltip: "URL of a website with information about the service",
+        infoUriPlaceholder: "The URL with information about this application",
+        infoUriTooltip: "URL of a website with information about the application",
         privacy_policy: "Privacy policy",
-        privacy_policyPlaceholder: "The Private policy URL  of the service",
+        privacy_policyPlaceholder: "The Private policy URL  of the application",
         privacy_policyTooltip: "A Privacy Policy is a legal requirement for all websites and apps that collect or use personal information from users.",
         accepted_user_policy: "Acceptable use policy",
-        accepted_user_policyPlaceholder: "The Acceptable Use Policy (AUP) of the service",
-        accepted_user_policyTooltip: "An acceptable use policy (AUP) is a document stipulating constraints and practices that a user must agree to for access to a service or system.",
+        accepted_user_policyPlaceholder: "The Acceptable Use Policy (AUP) of the application",
+        accepted_user_policyTooltip: "An acceptable use policy (AUP) is a document stipulating constraints and practices that a user must agree to for access to an application or system.",
```

Comment:
let's standardize the capitalization of Privacy Policy and Acceptable Use Policy here (dealer's choice)